tests/test_pvacseq_generate_aggregated_report.py

```diff
@@ -55,4 +55,6 @@ def test_compiles(self):
     def test_runs(self):
         input_file = os.path.join(self.test_data_directory, 'MHC_Class_I', 'Test.all_epitopes.tsv')
         output_file = tempfile.NamedTemporaryFile()
-        self.assertFalse(generate_aggregated_report.main([input_file, output_file.name]))
+        self.assertFalse(generate_aggregated_report.main([input_file, output_file.name, "--top-score-metric2=percentile"]))
+        self.assertFalse(generate_aggregated_report.main([input_file, output_file.name, "--top-score-metric2=ic50"]))
```

Comment:
It doesn't hurt anything to add these here but I don't think it's necessary. Same for the other tests/test_pvac*_generate_aggregated_report.py test.